tests/dev.yml

```diff
@@ -173,7 +173,7 @@ parameters:
                   appuio.io/node-class: "plus"
               note: "Will be scheduled on APPUiO Cloud plus nodes"
         minio:
-          enabled: false
+          enabled: true
```

Comment:
Please disable this again before merging, we don't need to deploy even more stuff in kindev if not necessary :D

Reply:
disabled